internal/fileutils/fileutils.go

```diff
@@ -265,6 +268,11 @@ func CopyFile(src, target string) error {
 	if err != nil {
 		return errs.Wrap(err, "out.Close failed")
 	}
+
```

Comment:
I found that, in two cases, I was recreating logic that was here simply because the file permissions were not be copied along with the files themselves.